test/jmh/BUILD

```diff
@@ -0,0 +1,6 @@
+load("//jmh:jmh.bzl", "scala_benchmark_jmh")
+
+scala_benchmark_jmh(
```

Comment:
Good thinking! Will specify some uselessly-low command line settings to make sure it doesn't take too long to run.